server/src/internalClusterTest/java/org/elasticsearch/action/admin/indices/create/ShrinkIndexIT.java

```diff
@@ -400,7 +401,7 @@ public void testCreateShrinkIndexFails() throws Exception {
 
         final InternalClusterInfoService infoService = (InternalClusterInfoService) internalCluster().getInstance(ClusterInfoService.class,
             internalCluster().getMasterName());
-        infoService.refresh();
+        ClusterInfoServiceUtils.refresh(infoService);
```

Comment:
Can we use `refreshClusterInfo()` instead to remove the line above as well?

Reply:
Ah yes, well spotted, see f16bd20e5aa33bc543e62fbf0c0a8af9fc696bf6.